source/options.tsx

```diff
@@ -10,6 +10,61 @@ import * as indentTextarea from 'indent-textarea';
 
 import {perDomainOptions} from './options-storage';
 
+function displayValidationStatus(text: string): void {
+	select('#validation')!.textContent = text;
+	select('form ul')!.classList.toggle('invalid', !text.includes('✔️'));
+}
+
+async function getTokenScopes(personalToken: string): Promise<string[]> {
+	const tokenLink = select('a#personal-token-link')!;
+	const url = tokenLink.host === 'github.com' ?
+		'https://api.github.com/' :
+		`${tokenLink.origin}/api/v3/`;
+
+	const response = await fetch(url, {
+		cache: 'no-store',
+		headers: {
+			'User-Agent': 'Refined GitHub',
+			Accept: 'application/vnd.github.v3+json',
+			Authorization: `token ${personalToken}`
+		}
+	});
+
+	if (!response.ok) {
+		const message = (await response.json())?.message ?? response.status;
```

Comment:
```suggestion
		const message = (await response.json())?.message
```

An I right that this is not even possible?

Reply:
I'd expect that a non-ok response would include an error, so yeah, we can drop `?.` too
```suggestion
		const {message} = (await response.json())!;
```